lib/i2s.toit

```diff
@@ -523,12 +523,30 @@ class Bus:
   /**
   Number of encountered errors.
 
-  If the program wasn't fast enough to read or write the buffers, then
-    a buffer overrun or underrun might occur. This is counted as an error.
+  This is the sum of $underrun-errors and $overrun-errors.
   */
   errors -> int:
+    return underrun-errors + overrun-errors
+
+  /**
+  Number of encountered underrun errors.
+
+  If the program wasn't fast enough to write the buffers, then
+    a buffer underun might occur. This is counted as an error.
+  */
+  underrun-errors -> int:
```

Comment:
I might be inclined to call this `errors-underrun` or even just have one `errors --overrun --underrun` method.

Reply:
done.